### ISSUE TITLE
MultiplayerDuplicator

### ISSUE THREAD

**Issue by Shadowblitz16** [godotengine/godot-proposals#4705]
### Describe the project you are working on

A spaceship game

### Describe the problem or limitation you are having in your project

Due to the fact I might not know what my node path is in the scene hierarchy I would like something like MultiplayerSpawner but it is used as a child of the node you want to spawn on the clients.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Basicly since the multiplayer duplicator node would be a child of the node you want to duplicate on the client sides
You wouldn't need to know the path where to spawn or what to spawn, you only need the root node you want to duplicate when it's created. 

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

![image](https://user-images.githubusercontent.com/2453707/174503725-61d9b164-8cd2-43db-8245-34303b855d43.png)

Basically you would supply a path to the node you want to duplicate in t5his case its the root of the scene..
Then when _ready is called in the MultiplayerDuplicator it would duplicate the specified root and add it to the root's parent...

```
@export var root_path : NodePath
[...]
func _ready():
    if multiplayer.is_server(): 
        var copy = get_node(root_path).duplicate()
        get_node(root_path).get_parent().add_child(copy)
```

### If this enhancement will not be used often, can it be worked around with a few lines of script?

Even though it is simple to implement our selves, It would make multiplayer easier and it would be a real built in type

### Is there a reason why this should be core and not an add-on in the asset library?

Its similar to MultiplayerSpawner

**Comment by Shadowblitz16** [comment]
I just realized this would be a bit more complicated then what I showed...
Basically it would have to not just spawn new instances but also sync up old ones...
Maybe This would be a better job for MultiplayerSynchronizer
